### DISCUSSION TITLE
コードブロックがリスト項目のときリストマーカーが表示されない

### DISCUSSION THREAD

**Discussion by yarakos95** [increments/qiita-discussions#354] [要望・アイデア・不具合]
リスト項目をコードブロックにした場合、リストマーカーは表示されなくなります。多くの場合、[デモ][demo] のようにコードブロックの左側にリストマーカーが表示されます。

[demo]: https://spec.commonmark.org/dingus/?text=-%0A%20%20%20%20%60%60%60python%0A%20%20%20%20print(%22Hello%20World%22)%0A%20%20%20%20%60%60%60%0A-%0A%20%20%20%20%20%20print(%22Hello%20World%22)%0A%0A-----%0A%0A1.%0A%20%20%20%20%60%60%60python%0A%20%20%20%20print(%22Hello%20World%22)%0A%20%20%20%20%60%60%60%0A1.%0A%20%20%20%20%20%20%20%20print(%22Hello%20World%22)%0A%0A

![image](https://user-images.githubusercontent.com/75878492/201599655-b2e878df-4dbf-4c69-bdf2-7be6c1d57221.png)

<details><summary>サンプルコード</summary>

```markdown
-
    ```python
    print("Hello World")
    ```
-
      print("Hello World")

-----

1.
    ```python
    print("Hello World")
    ```
1.
       print("Hello World")
```

</details>

### デモ

![image](https://user-images.githubusercontent.com/75878492/201600544-4e755263-4084-4033-bb2b-901be7ebacdf.png)

**Comment by tanaka-asami415** [comment]
@Yarakashi-Kikohshi 
ご報告ありがとうございます！
こちら調査の上、検討いたします🙇

**Comment by tsuzuki-takaaki** [comment]
@yarakos95
報告が遅くなってしまい、申し訳ございません。

こちらの不具合ですが、現在は解消されており、
リスト項目にコードブロックを指定した場合でも、リストマーカーが表示されるようになっております！

よろしくお願いします🙇

**Reply by yarakos95** [reply]
対応ありがとうございます！

実際に試したところ、リスト内のコードブロックにもリストマーカーが表示されることを確認しました 👀
